README.org

```diff
@@ -57,6 +57,34 @@ whether a directory is the project root:
   (setq pytest-project-root-test (lambda (dirname) (equal dirname "foo")))
 #+END_SRC
 
+By default, test paths passed to pytest are absolute paths:
+#+BEGIN_SRC sh
+  cd /project/root && pytest /base/path/relative/path/to/test.py
+#+END_SRC
+
+To override this behavior, set an optional source root directory:
+#+BEGIN_SRC elisp
+  (setq pytest-source-root-dir "/base/path/")
```

Comment:
Here I think we should keep this variable. Let me know what you think of this example. 

My code is in `~/Projects/myapp` and I mount things in docker under `/opt/myapp`. If I have `pytest-source-root-dir` (which could probably be named a little clearer but I don't have a better idea now) that means I can say I want replace the `WORKING-DIRECTORY` with the `pytest-source-root-dir` when figuring out the relative path. 

For example the `/project/root/path/to/test.py` using a `/opt/myapp` would replace the `/project/root` with `/opt/myapp`. An alternative is that you use can allow a `pytest-use-relative-test-paths` boolean that does the behavior you're defining with your existing code. 

Let me know if that makes sense and handles your use case clearly!

Reply:
If we're just solving the running-individual-tests-in-docker problem, then my preference would be to provide one simple solution to fixing test paths in the test environment when they don't match up with the absolute path in the local environment. For my use-case, I have no need to customize the local project root, but I could see how someone might want to do that (i.e. the case where docker-compose config is not at the project root). However, I wanted to avoid over-complicating the changes I'm making to this project unless users explicitly needed that additional functionality.

My understanding of pytest is that passing in a relative test path defined from the current working directory is equivalent to passing in a correct absolute path. Since that is the case, it is more durable to pass in the relative path, since any changes to your container's working directory would necessitate a change to any hardcoded container base path if you are setting an absolute path. 

I propose keeping it simple and sticking with the current setup of one additional optional string variable, which I should rename to something that makes more sense to you. Do you agree?